app/service/planning_svc.py

```diff
@@ -4,42 +4,49 @@
 from base64 import b64decode
 
 from app.objects.c_link import Link
-from app.utility.base_service import BaseService
+from app.utility.base_planning_svc import BasePlanningService
 from app.utility.rule import RuleSet
 
 
-class PlanningService(BaseService):
+class PlanningService(BasePlanningService):
 
     def __init__(self):
         self.log = self.add_service('planning_svc', self)
 
-    async def select_links(self, operation, agent, phase):
+    async def get_links(self, operation, agent, phase=None, trim=True):
```

Comment:
Let's make the `agent` variable optional (default to None) and have this function automatically include all links for all agents.  If an agent is passed, only return the links for that agent.